tezos-operation/src/lib.rs

```diff
@@ -142,7 +142,7 @@
 
 pub mod block_header;
 mod error;
-mod internal;
+pub mod internal;
```

Comment:
```suggestion
mod internal;
```
To reduce breaking changes in the future, this should be kept private.

Happy to discuss improvements for the API if you find the need to expose some internal method.